OpenRA.Mods.Common/AI/SupportPowerDecision.cs

```diff
@@ -94,7 +94,7 @@ public int GetAttractiveness(IEnumerable<Actor> actors, Player firedBy)
 		/// <summary>Makes up part of a decision, describing how to evaluate a target.</summary>
 		public class Consideration
 		{
-			public enum DecisionMetric { Health, Value, None }
+			public enum DecisionMetric { Health, Value, BaseBuilding, None }
```

Comment:
What is the new Metric for? Is it for targeting structures?

If so, this is already possible via the use of types.
Example:
```
Consideration@1:
   Against: Enemy
   Types: Structure
   Attractiveness: 1
   TargetMetric: None
   CheckRadius: 5c0
```